src/test/java/com/powsybl/sld/server/SingleLineDiagramTest.java

```diff
@@ -448,7 +450,7 @@ void testNetworkAreaDiagramWithValidConfig() throws Exception {
                 .voltageLevelToExpandIds(Collections.emptySet())
                 .voltageLevelToOmitIds(Collections.emptySet())
                 .positions(Collections.emptyList())
-                .withGeoData(false)
+                .nadGenerationMode(NadGenerationMode.PROVIDED_POSITIONS)
```

Comment:
I don't get the difference between PROVIDED_POSITIONS and CUSTOM_COORDINATES.
Which one means I want the basic Powsybl-diagram algorithm without provided positions ?